consutil/lib.py

```diff
@@ -75,16 +80,32 @@ def getBusyDevices():
             busyDevices[linenum_key] = (pid, date)
     return busyDevices
 
-# returns baud rate of device corresponding to line number
+# returns baud rate and flow control settings of device corresponding to line number
 # input: linenum (str)
-def getBaud(linenum):
-    checkDevice(linenum)
-    cmd = "sudo stty -F " + DEVICE_PREFIX + str(linenum)
-    output = run_command(cmd)
+def getConnectionInfo(linenum):
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    entry = config_db.get_entry(TABLE_NAME, str(linenum)) 
+    
+    baud_rate = "" if BAUD_KEY not in entry else entry[BAUD_KEY]
+    flow_control = False
+    if FLOW_KEY in entry and entry[FLOW_KEY] == "1":
+        flow_control = True
+
+    return (baud_rate, flow_control)
+
+# returns the line number of a given device name
+# input: devicename (str)
+def getLineNumber(devicename):
```

Comment:
is linenumber an integer, if yes, then return integer and if you cannot find a line number return None

Reply:
linenum is a string and if the line number isn't found, getLineNumber() returns an empty string. This is because in connect() the line number is only ever used as a string

Reply:
The comments for getLineNumber() and getConnectionInfo() should include the output type, I'll add it